web_external/js/views/body/AnalysisPanel.js

```diff
@@ -10,7 +10,7 @@
         return name  + 'Widget';
     };
 
-    minerva.views.AnalysisPanel = minerva.View.extend({
+    minerva.views.AnalysisPanel = minerva.views.Panel.extend({
```

Comment:
Maybe I'm misunderstanding, but if this were to happen it would have to be in the Panel.js initialize, which would mean extending panels would need to call parent initialize in their initialize. Is that what you're talking about?